pkg/backends/tcp.go

```diff
@@ -203,6 +203,7 @@ type TCPListenerCfg struct {
 	Port     int     `description:"Local TCP port to listen on" barevalue:"yes" required:"yes"`
 	TLS      string  `description:"Name of TLS server config"`
 	Cost     float64 `description:"Connection cost (weight)" default:"1.0"`
+	NodeCostMap map[string]float64 `description:"Connection cost (weight) for specific node"`
```

Comment:
Let's call this `NodeCosts` instead of `NodeCostMap` just for simplicity and readability in the config file.

Reply:
fixed